rakelib/vendor.rake

```diff
@@ -124,6 +124,33 @@ namespace "vendor" do
   end # task gems
   task "all" => "gems"
 
+  task "deps", [:gemfile] => [ "dependency:bundler" ] do |task, args|
```

Comment:
is this task name descriptive enough? this not only handle "deps" but  expands the specified